From bcf0c78f67e55939260b6fa33b6c54e3b6a1e115 Mon Sep 17 00:00:00 2001
From: Wenlin Kang <wenlin.kang@windriver.com>
Date: Wed, 11 Sep 2019 08:16:12 +0000
Subject: [PATCH] driver: pincrtl: pinctrl-imx: fix warning unused variable
 'config'

Remove unused variable 'config', fix below warning:
  warning: unused variable 'config' [-Wunused-variable]

This was introduced by commit 4db927320e724e6620d322ec01ba2f08bde016af
(driver: pincrtl: pinctrl-imx: remove attaching dev for regmap)

Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 drivers/pinctrl/freescale/pinctrl-imx.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/pinctrl/freescale/pinctrl-imx.c b/drivers/pinctrl/freescale/pinctrl-imx.c
index 9617d956db7f..88f6341837db 100644
--- a/drivers/pinctrl/freescale/pinctrl-imx.c
+++ b/drivers/pinctrl/freescale/pinctrl-imx.c
@@ -570,7 +570,6 @@ static void imx_free_resources(struct imx_pinctrl *ipctl)
 int imx_pinctrl_probe(struct platform_device *pdev,
 		      const struct imx_pinctrl_soc_info *info)
 {
-	struct regmap_config config = { .name = "gpr" };
 	struct device_node *dev_np = pdev->dev.of_node;
 	struct pinctrl_desc *imx_pinctrl_desc;
 	struct device_node *np;
-- 
2.17.1

